### ISSUE TITLE
wsrec: use of uninitialized value within wind_dirs_degs

### ISSUE THREAD

**Issue by sparvu** [kronometrix/recording#120]
wsrec during boot returns these error:

```
2016-11-12 06:57:05 info: started
2016-11-12 07:00:40 info: started
Use of uninitialized value in printf at /opt/kronometrix/bin/wsrec line 230.
Use of uninitialized value within @wind_dirs_degs in concatenation (.) or string at /opt/kronometrix/bin/wsrec line 283.
```

**Comment by sparvu** [comment]
in the raw data file we can see such values:

```
1478935555:61:99:14.4:2.7:2.6:2.7:0.0:1.1:W:-278.4:-278.4:278.4:959.5
1478935615:61:99:14.4:2.7:2.6:2.7:0.0:2.7:NW:-278.4:-278.4:278.4:959.5
```

**Comment by sparvu** [comment]
fixed in 1.1.2
